_posts/2015-09-14-we're-looking-for-an-experienced-manager-to-lead-our-UX-team.md

```diff
@@ -0,0 +1,73 @@
+
+Leisa Reichelt, Director of User Research for the UK’s [Government Digital Service](https://gds.blog.gov.uk/), wrote about the importance of this shared ownership of user experience in her article “[There is no UX, there is only UX](http://www.disambiguity.com/there-is-no-ux/)":
+
+>It is really important that no one in the team can point to someone over in the corner and put all the burden of user experience on that guy. No one person, no small group of people can be made responsible for the user experience of a service. It is down to the entire team to achieve this, and we need to drag people into the team who make decisions way before we get on the scene.
+
+Our UX designers, researchers, and content strategists add to this base understanding of UX through many means. We facilitate working sessions, wireframe designs, diagram complex systems, communicate functional specifications, plan user research, and work to maintain consistency across our products, platforms, and breakpoints. Jared Spool, founder of [UIE](http://www.uie.com/), recently wrote about this balance in “[Hiring UX Experts Versus Giving Your Team Their Own UX Skills](https://medium.com/uie-brain-sparks/hiring-ux-experts-versus-giving-your-team-their-own-ux-skills-c1fd9e4e480)”:
+
+>UX is a complex field because there are lots of complexities: accessibility, mobile design, working in government agencies, working in an agile development process, and dealing with cross-cultural needs, to name a few. Yet, these are edge cases. 80% of UX work is quite routine and when learned by everyone on the team, creates consistently good designs.
+
+What this means for our UX designers is:
+
+1.	We get to work alongside economists, attorneys, data analysts, and other experts in their fields to create tools, services, and regulations to help American consumers, and
+2.	We’re more able to focus on the complexity of designing for the entire American public in a marketplace known for fine print, confusing terms, and deceptive marketing.
+
+
+## What we’re doing 
+
+In four years, the CFPB has proposed new rules for mortgage servicing, prepaid cards, payday loans, money transfers, and debt collection. We’ve established partnerships with schools, libraries, and financial counselors to help educate and empower consumers. We’ve helped return over $10 billion to the American public as a result of our enforcement actions. As our team of designers and developers has grown, we’ve also launched several products and services to help advance the mission of the Bureau:
+
+* We’ve created print and online resources to help consumers [understand the home-buying process](http://www.consumerfinance.gov/owning-a-home/), [shop for a mortgage](http://www.consumerfinance.gov/owning-a-home/check-rates), and [understand the terms](http://www.consumerfinance.gov/owning-a-home/loan-options) before they sign.

```

Comment:
The "shop for a mortgage" link is now located at http://www.consumerfinance.gov/owning-a-home/explore-rates/